### PR TITLE
feat: group details screen (3.3)

### PR DESCRIPTION
**Pull request by OmarBasem**
Reverting some files from that PR: https://github.com/status-im/status-mobile/pull/14494

### CODE REVIEW

**Comment by OmarBasem** [comment]
@flexsurfer I have created another PR. I think it's all fixed now.

File: src/status_im/ui2/screens/chat/group_details/style.cljs [hunk 1]
```diff
@@ -33,12 +34,12 @@
...
 
-(def bottom-container
-  {:position           :absolute
-   :padding-horizontal 20
+(defn bottom-container
+  [safe-area]
+  {:padding-horizontal 20
    :padding-vertical   12
-   :padding-bottom     33
+   :padding-bottom     (+ 33 (if platform/ios? (:bottom safe-area) 0))
```
Comment (flexsurfer) [inline src/status_im/ui2/screens/chat/group_details/style.cljs]: why only ios ?

File: src/status_im/ui2/screens/chat/group_details/view.cljs [hunk 2]
```diff
@@ -66,44 +66,41 @@
...
-              (i18n/label :t/save)]]]))])))
+(defn add-members-sheet
+  [group admin?]
+  [:f>
+   (fn []
+     (let [{window-height :height} (rn/use-window-dimensions)
+           safe-area               (safe-area/use-safe-area)
+           selected-participants   (rf/sub [:group-chat/selected-participants])
+           deselected-members      (rf/sub [:group-chat/deselected-members])]
+       [rn/view {:style {:height (- window-height (:top safe-area))}}
```
Comment (flexsurfer) [inline src/status_im/ui2/screens/chat/group_details/view.cljs:77]: something wrong is here ,we shouldn't use height ,  and safe area is applied to all screens by default , so we shouldn't care about them in the screen itself

File: src/status_im/ui2/screens/chat/group_details/view.cljs [hunk 3]
```diff
@@ -1,11 +1,9 @@
 (ns status-im.ui2.screens.chat.group-details.view
```
Comment (flexsurfer) [inline src/status_im/ui2/screens/chat/group_details/view.cljs]: just wondering why we still make changes in ui2 namespace, can we move to status-im2 ?

File: src/status_im/ui2/screens/chat/group_details/view.cljs (hunk 2 above)
Comment (OmarBasem) [reply src/status_im/ui2/screens/chat/group_details/view.cljs:77]: That's a bottom sheet not a screen

File: src/status_im/ui2/screens/chat/group_details/view.cljs (hunk 3 above)
Comment (OmarBasem) [reply src/status_im/ui2/screens/chat/group_details/view.cljs]: Moved to status-im2

File: src/status_im/ui2/screens/chat/group_details/view.cljs (hunk 2 above)
Comment (flexsurfer) [reply src/status_im/ui2/screens/chat/group_details/view.cljs:77]: mhm

**Comment by OmarBasem** [comment]
@flexsurfer why keep the PR hanging? It keeps getting conflicts..

**Comment by flexsurfer** [comment]
> @flexsurfer why keep the PR hanging? It keeps getting conflicts..

what do you mean by keep the PR hanging?

**Review by flexsurfer — APPROVED** [review]

**Comment by OmarBasem** [comment]
> > @flexsurfer why keep the PR hanging? It keeps getting conflicts..
> 
> what do you mean by keep the PR hanging?

I mean not allowing the PR to move forward when it is complete
Anyways no problem

File: src/status_im2/subs/contact.cljs [hunk 4]
```diff
@@ -93,21 +93,20 @@
        vals)))
 
 (re-frame/reg-sub
- :contacts/sorted-and-grouped-by-first-letter
```
Comment (cammellos) [inline src/status_im2/subs/contact.cljs:96]: This is still used in some parts of the codebase in develop, and it's currently throwing an error, why has it been removed? is there a replacement to be used or is just a mistake?

Comment (OmarBasem) [reply src/status_im2/subs/contact.cljs:96]: @flexsurfer I think you shared with me the implementation of this method? Did you rename it by mistake?

**Pull request merged**